Rakefile

```diff
@@ -2,13 +2,23 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
-require File.expand_path('../config/application', __FILE__)
-require File.expand_path('../lib/tasks/evm_rake_helper', __FILE__)
+require 'bigdecimal'
+puts "Mem: #{((1024 * BigDecimal.new(`ps -o rss= -p #{Process.pid}`))/BigDecimal.new(1_048_576)).to_f}MiB"
```

Comment:
These two lines are tmp and can be ignored.  This is just how I got the mem vals in the examples.  Will be removed in the future.